tests/generic_config_updater/test_pfcwd_interval.py

```diff
@@ -163,6 +174,10 @@ def test_stop_pfcwd(duthost, ensure_dut_readiness):
                       "pfcwd unexpectedly still running on interface {}".format(interface))
     finally:
         delete_tmpfile(duthost, tmpfile)
+        # Restore default config
+        duthost.shell('config pfcwd stop')
```

Comment:
Why is this "restore default config" section necessary when we have the fixture also running `config pfcwd start_default`?

Reply:
@isabelmsft test_stop_pfcwd does not have proper clean up, and floods error messages to styslog. This influence all other TC and they started failing.